src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                       .filter(u -> u.getAge() != null)
+                       .filter(u -> (u.getAge() >= 30
+                                     && u.getAge() <= 45))
```

Comment:
```suggestion
                       .filter(u -> (u.getAge() >= 30
                                     && u.getAge() <= 45))
```
```suggestion
                       .filter(u -> (u.getAge() >= 30))
                       .filter(u -> u.getAge() <= 45)
```
필터를 두번 사용해보는 방법은 어떨까요~?